option optimize_for = LITE_RUNTIME;

message RebateInfoNotify
{
 required uint32 money = 1;
 required uint32 deadline = 2;
}
